core/src/components/chip/test/states/chip.e2e.ts

```diff
@@ -41,3 +41,23 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, screenshot, c
     });
   });
 });
+
+configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  /**
+   * This behavior only applies to Ionic Theme.
+   */
+  test.describe(title('chip: focus state'), () => {
+    test('should render focus state', async ({ page }) => {
```

Comment:
Should this be in a separate PR? I'm not sure I understand what this test has to do with verifying `shape` behavior. Also, I can't get the focus state using a keyboard so I'm not sure how this will work.

Reply:
@liamdebeasi About the focus, as the Chip by default has no events or interactions, the focus state is just there prepared in case a developer does it on their side, using also tabindex=0 for that effect.

Reply:
Or is there other way for these cases? Ate least, with :focus() or focus-visible() that would be my expectation on how this would work. Does the ion-focusable work in a different way?

Reply:
`ion-focusable` works similarly to `:focus-visible`, so the element needs to be eligible to receive a focus event for `ion-focused` to be applied.

Sounds good re: adding the tests.

Reply:
Ok, that was my expectation 👍. Then on the developer side they can add the tabindex=0 if they need it (it will be the case on the ODC Widget, where we have a Click event).